frameworks/Crystal/moonshine/server-redis.cr

```diff
@@ -30,6 +29,11 @@ end
 private def randomWorld
   id = rand(1..ID_MAXIMUM)
   num = REDIS.get("world:" + id.to_s)
+
+  if num == Nil
+    num = 7777
+  end
```

Comment:
Hi, Kain.

That code was my mock data to test crystal/moonshine on local machine.

I did not have redis on my local machine in that time, so just need to
pass redis test to see what will happen. (In reality, that if statement
will not happen)

Doing this I could figure out why crystal/moonshine fails. Redis problem
happens on travis causes the error; due to this all other frameworks
which fails redis test were resolved.

I just forgot to delete those line.

Now everything is correct and already merged.

No need to worry.

On 12/21/15 11:16 AM, Mike Smith wrote:
> In frameworks/Crystal/moonshine/server-redis.cr
> <https://github.com/TechEmpower/FrameworkBenchmarks/pull/1813#discussion_r48179437>:
> 
>> @@ -30,6 +29,11 @@ end
>>  private def randomWorld
>>    id = rand(1..ID_MAXIMUM)
>>    num = REDIS.get("world:" + id.to_s)
>> +
>> +  if num == Nil
>> +    num = 7777
>> +  end
> 
> This seems wrong to me - if you get a null pointer back from this, then
> there is something wrong with how we are getting the value for |id|, and
> we should definitely create special cases to deal with that.
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TechEmpower/FrameworkBenchmarks/pull/1813/files#r48179437>.
> 

-- 
Best,
Byung Lee